README.md

```diff
@@ -57,30 +57,50 @@ LGPL code.
 
 RIOT provides features including, but not limited to:
 
-* a preemptive, tickless scheduler with priorities
-* flexible memory management
-* high resolution, long-term timers
-* MTD abstraction layer
-* File System integration
-* support 200+ boards based on AVR, MSP430, ESP8266, ESP32, RISC-V,
-  ARM7 and ARM Cortex-M
-* the native port allows to run RIOT as-is on Linux and BSD.
-  Multiple instances of RIOT running on a single machine can also be
-  interconnected via a simple virtual Ethernet bridge or via a simulated
-  IEEE 802.15.4 network (ZEP)
-* IPv6
-* 6LoWPAN (RFC4944, RFC6282, and RFC6775)
-* UDP
-* RPL (storing mode, P2P mode)
-* CoAP
-* OTA updates via SUIT
-* MQTT
-* USB (device mode)
-* Display / Touchscreen support
-* CCN-Lite
-* LoRaWAN
-* UWB
-* Bluetooth (BLE) via [NimBLE](https://github.com/apache/mynewt-nimble)
+ * Platforms Supported
+    - 200+ boards based on AVR, MSP430, ESP8266, ESP32, RISC-V, ARM7, and ARM Cortex-M
+    - the native port allows to run RIOT as-is on Linux and BSD.
+      Multiple instances of RIOT running on a single machine can also be
+      interconnected via a simple virtual Ethernet bridge or via a simulated
+      IEEE 802.15.4 network (ZEP)
+
+ * Programming Environment
+    - Write native code in C, C++, or Rust
+    - Rely upon POSIX APIs, user friendly hardware abstractions, and standard embedded Rust interfaces
+    - Integrate dynamic runtimes such as MicroPython, elm (JavaScript), and WASM
+
+ * Network Features and Protocols
+    - IPv6
+    - 6LoWPAN (RFC4944, RFC6282, and RFC6775)
+    - UDP
+    - RPL (storing mode, P2P mode)
+    - CoAP
+    - MQTT
+    - Bluetooth (BLE) via [NimBLE](https://github.com/apache/mynewt-nimble)
+    - LoRaWAN
+    - UWB
+    - CNN-Lite
+    - LwIP
+    - Dose (single pair ethernet like networking on serial interface UART)
+
+ * System Features
+    - a preemptive, tickless scheduler with priorities
+    - flexible memory management
```

Comment:
Which memory management are we referring to here?

Reply:
probably the one that does not force you to have a heap available but can use it if you need it -- my guess

Reply:
let's not add things that we are not sure what they mean ourself :sweat_smile:

Reply:
It is, as of now, the 2nd most prominent feature in the list.

I also don't know what is meant by that and I do agree that it should be dropped if nobody can explain it.

But maybe `git praise` and ask the author first?

Reply:
This line was introduced with the README.md 29.12.12 @784ef523900f3b0942c43099f57621b30f6cab73 - it is in it current form the one of the second oldest lines in this file (due to a typo fix) -- if the git history wasn't manipulated.
Its original author does already know that this question exists.

Reply:
@OlegHahm: Ist das Kunst oder kann das weg? :wink:

Reply:
Wow, interesting... The only thing I can imagine that I wanted to express that in RIOT you can actually have multiple stacks and can use malloc() (TLSF was introduced early on) in contrast to Contiki. But yeah, please remove!

Reply:
seems like that statement has not become untrue
```suggestion
    - flexible memory management (TLSF real-time capable pooled heap memory integration available via pkg)
```

Reply:
side: at first i read tlsf as tls - thread local storage:  to my knowlege we dont have that (well supported) libc needs to feature that and only than we could add the supporting bits to the scheduler and even than one has to be very careful since our exec might not happen in the thread that is tasked with a specific function.

Reply:
if i did not misread tlsf-malloc replaces malloc if that pkg is used, so it has not to used explicitly.

According to the initial paper it has a lower complexity class O(1) than first fit alloc (I am assuming embedded libc take the easy route and did not check),  pooling is also an advantage (more safe) in that the malloc fails (at least that can be handled) rather then letting the heap run into the main stack.

Reply:
- the malloc fails (at least that can be handled) rather then letting the heap run into the main stack

that's what any sane malloc implementation would do, `tests/sys/malloc` will rely on that

 - can work with multiple memory regions

so do [newlib](https://github.com/RIOT-OS/RIOT/blob/master/sys/newlib_syscalls_default/syscalls.c#L117)/[picolibc](https://github.com/RIOT-OS/RIOT/blob/master/sys/picolibc_syscalls_default/syscalls.c#L83)

I think this is a pretty niche feature, no need to advertise it so prominently

Reply:
The main reason to add TLSF in the first place was its real-time capability (because of O(1)).